<ui version="4.0" >
 <class>QmitkLevelWindowPresetDefinition</class>
 <widget class="QDialog" name="QmitkLevelWindowPresetDefinition" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>281</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>610</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Preset definition</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <widget class="QTableView" name="presetView" >
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid" >
      <bool>false</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="presetnameLabel" >
         <property name="text" >
          <string>Preset:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2" >
        <widget class="QLineEdit" name="presetnameLineEdit" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="levelLabel" >
         <property name="text" >
          <string>Level:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QSpinBox" name="levelSpinBox" >
         <property name="minimum" >
          <number>-1024</number>
         </property>
         <property name="maximum" >
          <number>3072</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="windowLabel" >
         <property name="text" >
          <string>Window:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QSpinBox" name="windowSpinBox" >
         <property name="maximum" >
          <number>4096</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QPushButton" name="addButton" >
         <property name="text" >
          <string>Add preset</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QPushButton" name="changeButton" >
         <property name="text" >
          <string>Change</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" >
        <widget class="QPushButton" name="removeButton" >
         <property name="text" >
          <string>Delete preset</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>91</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>204</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton" >
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>changeButton</tabstop>
  <tabstop>presetnameLineEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>QmitkLevelWindowPresetDefinition</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>460</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>QmitkLevelWindowPresetDefinition</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>545</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
